src/state/image-dataset/types.ts

```diff
@@ -21,17 +21,17 @@ export interface DatasetMetaData {
     version: string;
     id: string;
     description: string;
-    image: string;
+    image?: string;
     index: number;
     link?: string;
     manifest?: string;
     production?: boolean;
     userData: {
-        isNew: boolean;
-        inReview: boolean;
-        totalTaggedStructures: number;
-        totalCells: number;
-        totalFOVs: number;
+        isNew?: boolean;
+        inReview?: boolean;
+        totalTaggedStructures?: number;
+        totalCells?: number;
+        totalFOVs?: number;
     };
```

Comment:
Should `userData` also be allowed to be undefined?

Reply:
Seems like a valid question for overall strategy, but the `&&` conditionals that look at the properties of userData in `DatasetCard` will throw errors if you make this optional now
`userData.totalTaggedStructures && <Descrip...`